src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+let rightButtonClicked = false;
+let leftButtonClicked = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    leftButtonClicked = true;
+    resolve('First promise was resolved');
+  });
+
+  if (!leftButtonClicked) {
+    setTimeout(() => {
+      // eslint-disable-next-line
+      reject(`First promise was rejected`);
+    }, 3000);
+  }
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
```

Comment:
The second promise resolves on any left or right mouse button press. Ensure this behavior aligns with the intended functionality, as it might resolve too easily.